apps/web/app/(booking-page-wrapper)/org/[orgSlug]/[user]/page.tsx

```diff
@@ -67,7 +67,8 @@ export const generateMetadata = async ({ params, searchParams }: PageProps) => {
         () => profile.name,
         () => markdownStrippedBio,
         false,
-        getOrgFullOrigin(entity.orgSlug ?? null)
+        getOrgFullOrigin(entity.orgSlug ?? null),
+        `/${decodedParams.user}`
```

Comment:
- pathname is manually constructed here only because it's needed for og:url metadata (that's the only place it's really needed)